bin/periphery/src/api/mod.rs

```diff
@@ -59,6 +59,11 @@ pub enum PeripheryRequest {
   #[to_string_resolver]
   GetSystemProcesses(GetSystemProcesses),
   GetLatestCommit(GetLatestCommit),
+  // Networks Stats (Read)
+  // #[to_string_resolver]
+  // GetNetworkStatsByInterface(GetNetworkStatsByInterface),
+  // #[to_string_resolver]
+  // GetTotalNetworkStats(GetTotalNetworkStats),
```

Comment:
These methods aren't needed, much like methods specifically to get disk usage aren't needed. This commented out stuff can be cleaned up.

Reply:
Done